woocommerce/woocommerce-android
# [Google for Woo] Release i1

Part of: #11790


### Description


This PR releases the Google for Woo i1 feature to public! 🥳 🤞🏼 

It also bumps the plugin requirement to be at least 2.7.7, as that version has an important fix to make the campaign form looks correct on mobile.


| Before | After |
|-|-|
| ![image](https://github.com/user-attachments/assets/8bbe224a-aa56-4218-83dc-6db54e449693) | ![image](https://github.com/user-attachments/assets/af634282-9f80-4c08-af47-abdf69863bd8) |


### Testing instructions
1. Make sure the Google Listings and Ads plugin is updated to 2.7.7. If you're using one of my test sites, they have all been updated
2. Open campaign creation (easiest way might be to use MoreMenu, edit `MoreMenuViewModel.determineUrlToOpen()` to always return  `baseUrl.slashJoin(AppUrls.GOOGLE_ADMIN_CAMPAIGN_CREATION_SUFFIX)`
3. Check that the design now looks like the result above. The important part is that the left and right side don't have cut off text anymore.


No other new functionality is added so aside from checking the campaign creation form above, nothing but smoke testing is needed. Thank you!
